bears/java/CheckstyleBear.py

```diff
@@ -33,7 +33,7 @@ def known_checkstyle_or_path(setting):
 
 @linter(executable='java',
         output_format='regex',
-        output_regex=r'\[(?P<severity>WARN|INFO)\].*?'
+        output_regex=r'\[(?P<severity>WARN|INFO|ERROR)\].*?'
```

Comment:
Good catch.  It seems (`git grep '<severity>'`) ERROR|WARN|INFO would be the more common ordering.